src/PersonReader.java

```diff
@@ -1,6 +1,66 @@
-public class PersonReader {
-    public static void main(String[] args) {
-        // Write your code here ... ⬇
 
+    import java.awt.*;
+    import java.util.Scanner;
+    import javax.swing.JFileChooser;
+    import java.io.FileNotFoundException;
+    import java.io.File;
+    import java.util.StringTokenizer;
+
+
+    public class PersonReader
+{
+
+
+
+    public static void main(String[] args)
+    {
+
+        boolean hasFile = false;
+        File selectedFile = new File("");
+        JFileChooser choose = new JFileChooser();
+        String filename = "classmates.csv";
+        String lines = "";
+        String currToken = "";
+
+
+
+        choose.setCurrentDirectory(new File("C:\\Users\\winso\\OneDrive\\Documents\\UC"));
+
+        do {
+            if (choose.showOpenDialog((Component)null) == 0) {
+                selectedFile = choose.getSelectedFile();
+                hasFile = true;
+            } else {
+                System.out.println("You must select a file! \n Try again.");
+            }
+        } while(!hasFile);
+
+        try
+        {
+            Scanner peopleFile = new Scanner(selectedFile);
+
+            System.out.println("Firstname  Lastname    Age         email");
+            System.out.println("=================================================");
+
+            while(peopleFile.hasNext())
+            {
+                lines = peopleFile.nextLine();
+                StringTokenizer tokenizer = new StringTokenizer(lines,"," );
+                    while(tokenizer.hasMoreTokens())
+                    {
+                        currToken = tokenizer.nextToken();
+                        System.out.print(String.format("%-12s", currToken));
+                    }
+                System.out.println("\n");
```

Comment:
this is making 2 new lines instead of one. you could do either this 
```suggestion
                System.out.print("\n");
```
or that
```suggestion
                System.out.println();
```